### PR TITLE
[WIP] Victim with ID

### PR DESCRIPTION
**Pull request by esarafianou**

### CODE REVIEW

File: backend/breach/views.py [hunk 1]
```diff
@@ -29,6 +29,9 @@ def get_work(request, victim_id=0):
     except:
         raise Http404('Victim not found')
 
+    if victim.state == 'paused':
+        raise Http404("No work for the victim")
```
Comment (dionyziz) [inline backend/breach/views.py]: quote consistency

Comment (dionyziz) [inline backend/breach/views.py]: This mandates a more meaningful response message for easier debugging. How about?

```'The attack for victim %i is paused' % victim.id```

File: backend/breach/views.py [hunk 2]
```diff
@@ -160,3 +162,11 @@ def victimID(request, victim_id=0):
             victim.state = 'running'
         victim.save()
         return HttpResponse(status=200)
+    elif request.method == 'DELETE':
+        victim = Victim.objects.get(pk=request.DELETE['victim_id'])[0]
+	if not victim.trashed_at:
+	    Victim.delete(victim)
+	    Timer(10, Victim.delete(victim), ()).start()
+	if victim.trashed_at:
```
Comment (dimkarakostas) [inline backend/breach/views.py]: use "else"?

File: backend/breach/views.py [hunk 3]
```diff
@@ -160,3 +162,11 @@ def victimID(request, victim_id=0):
...
+            Victim.restore(victim)
+	    Timer(10, Victim.restore(victim), ()).start()
```
Comment (dionyziz) [inline backend/breach/views.py]: nit indent

File: backend/breach/views.py [hunk 4]
```diff
@@ -160,3 +162,11 @@ def victimID(request, victim_id=0):
             victim.state = 'running'
         victim.save()
         return HttpResponse(status=200)
+    elif request.method == 'DELETE':
+        victim = Victim.objects.get(pk=request.DELETE['victim_id'])[0]
+	if not victim.trashed_at:
+	    Victim.delete(victim)
+	    Timer(10, Victim.delete(victim), ()).start()
```
Comment (dionyziz) [inline backend/breach/views.py]: No need for timers – just leave stuff in the trash. Make sure you don't treat trashed victims as regular victims in operations such as get_work though.

**Comment by esarafianou** [comment]
Merged with PR for Victim & Target

**Pull request closed without merging**